gips/0081-indexing-payments.md

```diff
@@ -0,0 +1,165 @@
+---
+GIP: 0081
```

Comment:
I believe needed in YAML parsing to avoid being treated as 81 and leading zeros dropped.
```suggestion
GIP: 0081
```
```suggestion
GIP: "0081"
```